File: SOFTWARE/Models/Turno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace SOFTWARE.Models
{
    public class Turno
    {
        
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Numero { get; set; }

        public string Motivo { get; set; }

        public string Asistencia { get; set; }

        public string DescripcionOperacion { get; set; }

        public string ContratistaAtendio { get; set; }

        public string RefTiempo { get; set; }

        public string RefSolicitante { get; set; }

        public string FechaFinalizacion { get; set; }
        
    }
}

File: SOFTWARE/ClientApp/src/app/Modelo/turno.ts
export class Turno {

  numero:number;

  motivo:string | null;

  asistencia:string | null;

  descripcionOperacion:string | null;

  contratistaAtendio:string | null;

  refTiempo:string;

  refSolicitante:string | null;

  fechaFinalizacion: string;

  poblacion: string | null;

  observacion: string | null;

}

export class TurnoHistorico extends Turno{
  nombreSolicitante: string | null;
}
